plugins/modules/helm.py

```diff
@@ -677,6 +678,9 @@ def main():
     if update_repo_cache:
         run_repo_update(module, helm_cmd_common)
 
+    if kubeconfig:
```

Comment:
@suppaduppax Thanks for the contribution. This is already handled in [here](https://github.com/ansible-collections/kubernetes.core/blob/b54e9ef4ef36e8da5a9a30b0c2d39232c2773aa8/plugins/module_utils/helm.py#L48). So I think this check is irrelevant.